patches/mitt@3.0.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/package.json b/package.json
+index ad38c7bd3a6eef608058259f8b2523b3a12b935d..ae9dc6f91a216aaa793294036443dbd839e3375d 100644
+--- a/package.json
++++ b/package.json
+@@ -9,6 +9,7 @@
+   "source": "src/index.ts",
+   "typings": "index.d.ts",
+   "exports": {
++    "types": "./index.d.ts",
```

Comment:
Aren't we staying with root "types" field?

Reply:
Root `types` will be used in the `-sass` repository. In this one, we've switched to `moduleResolution: bundler` and with that when `exports` are present in any package they become the only source of information, so packages in `node_modules` have to have valid `exports`